src/USER-REAXC/reaxc_ffield.cpp

```diff
@@ -158,15 +150,9 @@ char Read_Force_Field( FILE *fp, reax_interaction *reax,
     c = Tokenize( s, &tmp );
 
     /* Sanity checks */
-    if (c == 2 && !lgflag) {
```

Comment:
why delete this here? to provide this specific error message and not some generic "inconsistent ffield file" this check has to come first.

Reply:
I actually moved it below (around lines 314-318), because this check wasn't triggered with my test potential with only one element, but this was a mistake - both checks are necessary. I'll restore it.